src/components/common/Select.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Box, InputLabel, MenuItem, Select } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(() => ({
+  input: {
+    width: 328,
```

Comment:
I noticed this is also used for `TextInput`. Do we wanna move this width value into our constants file or maybe theme file?

Reply:
ack realized that we prob don't want this constant at all, since the sidebar inputs are narrower

changed this to a prop instead, defined a constant in `FormFieldGroup` to pass in :)